python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -402,7 +402,11 @@ def setDefaults(self):
 
         # Only reject sky sources; we already selected good stars.
         self.astrometry.sourceSelector["science"].doFlags = True
-        self.astrometry.sourceSelector["science"].flags.bad = ["sky_source"]
+        self.astrometry.sourceSelector["science"].flags.good = ["calib_psf_candidate"]
+        self.astrometry.sourceSelector["science"].flags.bad = []
```

Comment:
Why is `"sky_source"`  removed from the bad flag list? I imagine if someone overrides the good flag list, they would want sky sources removed no matter what.

Reply:
Because the sky sources aren't run at this stage.  I removed it because it caused an error due to this column not existing.